src/codeu/chat/common/ServerInfoTest.java

```diff
@@ -0,0 +1,20 @@
+package codeu.chat.common;
+import static org.junit.Assert.assertEquals;
+import org.junit.Test;
+import codeu.chat.util.Uuid;
+
+import java.io.IOException;
+
+public class ServerInfoTest {
+
+    @Test
+    public void testServerInfo() {
+        try{
+            ServerInfo server2 = new ServerInfo(Uuid.parse("1.0.1"));
+            assertEquals("1.0.1", server2.toString());
+        }
+        catch (IOException e){
+            System.out.println("Invalid input");
```

Comment:
I added three more tests, two which should return an Exception, and one with a negative version